File: src/main/java/com/kika/smllybot/database/sql/users/dto/UserAccount.java
package com.kika.smllybot.database.sql.users.dto;

import java.time.OffsetDateTime;

public record UserAccount(long id, long discordId,
                          String role,
                          String name,
                          String motto,
                          OffsetDateTime createdAt,
                          int reaction,
                          Long citizenship,
                          OffsetDateTime citizenshipData
) {}

File: src/main/java/com/kika/smllybot/database/sql/users/dto/UserAccount.kt
package com.kika.smllybot.database.sql.users.dto

import java.time.OffsetDateTime

data class UserAccount(
    val id: Long,
    val discordId: Long,
    val role: String?,
    val name: String,
    val motto: String?,
    val createdAt: OffsetDateTime,
    val reaction: Long,
    val citizenship: Long?,
    val citizenshipData: OffsetDateTime?
)
